src/chigame/games/models.py

```diff
@@ -517,3 +517,25 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return "Message from " + str(self.sender) + ": " + self.content
+
+
+class Review(models.Model):
+    "Represents a game review"
+    title = models.TextField(blank=True, null=True)
+    review = models.TextField(blank=True, null=True)
+    rating = models.PositiveSmallIntegerField(blank=True, null=True)
```

Comment:
What is the expected range of the rating? Is it 1-5, 1-100? Something else?
If you have a specific one in mind please add a code comment **and also validation** that check whether the rating is in the correct range.

Reply:
I've added validation to the `rating` field and fixed the issue where blank rating and review fields were allowed.
![Screenshot 2023-11-30 at 5 41 25 AM](https://github.com/uchicago-cs/chigame/assets/81624602/5bd00cb0-4062-4bdf-a722-a772a7c2734d)
I added a small comment under `rating`. I wasn't sure if that's what you meant by documentation or if there's another place/way I should go about it. Thanks!